python/ml4ir/applications/ranking/tests/test_fixed_additive_positional_bias.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+import warnings
+import tensorflow as tf
+from ml4ir.base.model.architectures.fixed_additive_positional_bias import FixedAdditivePositionalBias
+import numpy as np
+
+warnings.filterwarnings("ignore")
+
+
+class TestFixedAdditivePositionalBias(unittest.TestCase):
```

Comment:
@mohazahran Can you also add a test for the main `call()` method. I recommended moving the one hot inside the `call()` method itself at another place.